Frends.SMTP.SendEmail/Frends.Smtp.SendEmail.Tests/SendEmailTests.cs

```diff
@@ -42,7 +42,7 @@ public void EmailTestSetup()
             _filepath = Path.Combine(_localAttachmentFolder, TEST_FILE_NAME);
 
             if (!File.Exists(_filepath))
-                using (File.Create(_filepath)) { }
```

Comment:
Right, so what happens now that the using is without its code block? In the code that had the `{ }` made it clear that we get the FileStream returned by calling File.Create and dispose it. Now without the `{ }` - where does the using block end?..

Reply:
changed back to {} and added pragma so it won't throw warning